CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ This file is used to list changes made in each version of the AWS ParallelCluste
   EFS is used for shared internal data. 
 - On Ubuntu systems, remove default logrotate configuration for cloud-init log files that clashed with the
   configuration coming from Parallelcluster.
+- Renaming pcluster.sh profile to cookbook_environment.sh profile which will be used only for cfn commands.
```

Comment:
Here we should explain the change from the user perspective. What's the consequence of this change for the user?